ui/src/App.vue

```diff
@@ -77,11 +77,14 @@ export default class App extends Vue {
 </script>
 
 <style>
+html {
+  overflow: auto !important;
+}
+
 #app {
   font-family: 'Avenir', Helvetica, Arial, sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  text-align: center;
```

Comment:
Probably quite nitpicky, but I see you removed this rule to stop the text in the JSON editor to be centered.

Now the menu is not centered anymore (quite minimal change, but i actually liked the centered menu).

Reply:
I actually intended to left align the menu items and headings here. The editor was just a biproduct of the cascading css.

The convention, when looking at [Vuetify](https://vuetifyjs.com/en/components/), [Vue Docs](https://vuejs.org/v2/guide/), [Angular Docs](https://angular.io/start), is to align the menu items and headings on the left. It's really up to you but conventions are always something valuable when it comes to web-usability.

Reply:
I agree, then I guess the way we did it before was unconventional. Let's stick to the left alignment then.